pkg/ddl/placement_policy.go

```diff
@@ -183,7 +184,11 @@ func checkAllTablePlacementPoliciesExistAndCancelNonExistJob(t *meta.Meta, job *
 }
 
 func onDropPlacementPolicy(jobCtx *jobContext, t *meta.Meta, job *model.Job) (ver int64, _ error) {
-	policyInfo, err := checkPlacementPolicyExistAndCancelNonExistJob(t, job, job.SchemaID)
+	args, err := model.GetPlacementPolicyArgs(job)
+	if err != nil {
+		return ver, errors.Trace(err)
+	}
+	policyInfo, err := checkPlacementPolicyExistAndCancelNonExistJob(t, job, args.PolicyID)
```

Comment:
Why need to use args.PolicyID replace job.SchemaID ?

Reply:
Just want to put all related variables into arguments. I think using `job.SchemaID` to store `PolicyID` is a bit confusing.